### PR TITLE
[N/A] Blotter/Chart launch fix

### PR DESCRIPTION
**Pull request by ottah**
- Fixes launching Chart apps from the context menu.
- Adds typed payload for context menu.

### CODE REVIEW

File: src/demo/components/common/ContextMenu.ts
Comment (pjbroadbent) [inline src/demo/components/common/ContextMenu.ts]: Is `ContextMenuPayload` now unused? Can it be removed?

File: src/demo/components/common/ContextMenu.ts
Comment (pjbroadbent) [inline src/demo/components/common/ContextMenu.ts]: Not checked out this branch, but are there still style updates pending?

Last time I saw the context menus, was no indication of which items expanded into sub-menus.

Comment (ottah) [reply src/demo/components/common/ContextMenu.ts]: I'll add that now.

File: src/demo/components/blotter/SymbolsRow.tsx
Comment (bryangaleopenfin) [inline src/demo/components/blotter/SymbolsRow.tsx]: Suggest use AppName from directory.ts here

**Review by bryangaleopenfin — APPROVED** [review]
Looks good. Approved with comment

**Review by pjbroadbent — APPROVED** [review]

**Review by tomrobertsOF — APPROVED** [review]
Works well and looks great! There's some conflicts (I think just linting), but approved pending those being resolved.

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@ottah). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If there are any questions, please don't hesitate to [get in touch with our Infrastructure Support team](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/77955298/Getting+in+Touch#GettinginTouch-InfrastructureSupport). /CC @finos-admin

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@ottah). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If there are any questions, please don't hesitate to [get in touch with our Infrastructure Support team](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/77955298/Getting+in+Touch#GettinginTouch-InfrastructureSupport). /CC @finos-admin

**Pull request closed without merging**
